spec/factories/users.rb

```diff
@@ -5,7 +5,6 @@
     password      { Faker::Internet.password(min_length: 8) }
     gender        { User.genders.values.sample }
     provider      { 'email' }
-    uid           { Faker::Internet.uuid }
```

Comment:
Why this line got deleted?

Reply:
UID is calculated by the devise gem. If the provider was set to email, then the UID is automatically set to the email address. e.g. when a user signs-up with the email: `test@test.com`, then the UID is `test@test.com`.

Reply:
mmm... but i think it can be faked anyway, otherwise users will be create a user with uid in blank or nil